kedro/template/{{ cookiecutter.repo_name }}/kedro_cli.py

```diff
@@ -249,6 +253,11 @@ def build_docs():
         ]
     )
     call(["sphinx-build", "-M", "html", "docs/source", "docs/build", "-a"])
+    if open_docs:
+        p = Path("docs/build/html/index.html").resolve().as_uri()
+        secho("Opening {}".format(p))
+        webbrowser.open(str(p))
```

Comment:
Can we replace "p" here with a more descriptive variable name? :) "docs_page", "docs_home" or anything like that would suffice.

Reply:
Also I think the return of `as_uri()` is a `string` anyway, so no need for casting here. If you want to be on the safe side regardless, you could add it to line 257, so that would get passed through to `format` and `open` calls.

Reply:
Thanks for the feedback, super valuable! 🚀

Should be fixed now. :)